src/install/install.zig

```diff
@@ -1366,7 +1366,7 @@ pub const PackageInstall = struct {
             cached_package_dir,
             this.allocator,
             &[_]bun.OSPathSlice{},
-            &[_]bun.OSPathSlice{bun.OSPathLiteral("node_modules")},
```

Comment:
TIL you can publish an npm package with a `node_modules` folder as long as it isn't top level